tools/java/cpp-build/pom.xml

```diff
@@ -44,16 +64,42 @@
             </goals>
             <configuration>
               <sources>
+                <!-- Add generated/ to the source directories. -->
+                <!-- TODO: Add needed functionality to java/libphonenumber/src/Phonemetadata.java
```

Comment:
I wouldn't say necessarily "add needed functionality" because that doesn't solve the problem of it potentially getting out of sync.. Also depending on a maven release might be fine if we specified the version and said the C++ one had to be the same. I would state the problem and leave it open in the TODO how we are going to solve it.